CONTRIBUTING.md

```diff
@@ -77,10 +77,10 @@ Tests that interact with cloud providers are written using a replay method,
 where the test is run and actually performs the operations, and the
 requests/results of the operations are stored in a replay file. This replay file
 is then read back in unit tests on Travis, so the tests get to operate with real
-data. Unfortunately, while the Go Cloud team can generate these replay files
-against our test cloud infrastructure, it is not yet possible for external
-contributors to do the same. We want to improve this process in the future and
-are researching how we can do this. If you have any ideas, please file an issue!
+data. Unfortunately, while the GCDK team can generate these replay files against
```

Comment:
Should we use Go CDK vs. GCDK consistently? Which one?

Reply:
Good catch; yes. I had been using GCDK earlier and didn't fix these. Done.